lib/pages/home/wallet/wallet.dart

```diff
@@ -57,12 +62,19 @@ class WalletPage extends StatelessWidget {
     );
   }
 
-  Widget _buildCurrencyRow(BuildContext context, Currency currency) {
+  Widget _buildCurrencyRow(BuildContext context, int index, Currency currency) {
+    Color currencyRowColour = Colors.white12;
+    Color highlightColor = Theme.of(context).brightness ==  Brightness.light ? Colors.black12 : Colors.white12;
+    currencyRowColour = (index % 2 != 0 ? currencyRowColour = highlightColor : currencyRowColour = Colors.transparent);
     return Container(
-      margin: EdgeInsets.all(8.0),
+      margin: EdgeInsets.all(0.0),
```

Comment:
This line can be removed because you set the margin to 0, which is the default.

Reply:
Updated